docs/modeling/connectors/ootb/content.md

```diff
@@ -0,0 +1,367 @@
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `useFiles` | *Optional.* Set to `true` to include files in the search query.  | Boolean | True |
+| `useFolders`| *Optional.* Set to `true` to include folders in the search query. | Boolean | False | 
+
+### Select a file
+The `SELECT_FILE` action is used to select a file from the Alfresco Content Services repository to be used in the process.
+
+No additional input parameters are required to select a file. The following output parameter is included that contains the selected file: 
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `files` | *Required.* The files returned as a [file](../../files.md) variable. | File | orders-file | 
+
+### Select a folder
+The `SELECT_FOLDER` action is used to select a folder from the Alfresco Content Services repository to be used in the process.
+
+No additional input parameters are required to select a folder. The following output parameter is included that contains the selected folder: 
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `folders` | *Required.* The folder returned as a [process variable](../../processes/variables.md) of type folder. | Folder | 2020-shipments | 
+
+### Select content metadata
+The `SELECT_METADATA` action is used to select the metadata from a file or folder from Alfresco Content Services.
+
+| Parameter | Description | Type | Example |
+| --------- | ----------- | ---- | ------- |
+| `useFiles` | *Optional.* Set to `true` to include files in the search query.  | Boolean | True |
+| `useFolders`| *Optional.* Set to `true` to include folders in the search query. | Boolean | False | 
+
+The following is an example response of the selected metadata:
+
+```json
+{
+	"164e0082-5ac1-4afa-a09b-619a8c79f66f": {
+		"cm:categories": [],
+		"cm:lastThumbnailModification": [
+			"imgpreview:1588671895595"
+    ]
+  }
+}
+```
+
+## Events
+The following events can be configured as event [triggers](../../triggers.md) using the content connector:
+
+* [Content created](#content-created)
+* [Content updated](#content-updated)
+* [Content moved](#content-moved)
+* [Content deleted](#content-deleted)
+
+### Content created
+This event is fired when a file or folder is created in Alfresco Content Services that matches the input parameters defined in the trigger event.
+
+For a file the event is called `FILE_CREATED`. 
+
+For a folder the event is called `FOLDER_CREATED`.
+
+#### Input parameters
+
+| Parameter | Description | Type | Example |
+| --------  | ----------- | ---- | ------- |
+| `nodeName` | The name of the node that is created. | String | 2020-orders | 
+| `parentFolder` | The node ID of the parent folder. | String | dioos2a1-2a22-4f42-820d-c2b628751121 | 
+| `nodeType` | The content type of the node that is created. | String | cm:file | 
+
+#### Output parameters
```

Comment:
Output parameters are common to all the events, so if you prefer you can write it only once as it is done in the connector action with the input parameters. (up to you)

Reply:
This makes it a lot more readable - good suggestion! 👍